src/System.Management.Automation/engine/CommandCompletion/CompletionAnalysis.cs

```diff
@@ -905,7 +905,7 @@ internal List<CompletionResult> GetResultHelper(CompletionContext completionCont
                         {
                             switch (completionContext.TokenBeforeCursor.Kind)
                             {
-                               
```

Comment:
Can we run `dotnet-format` on the repo to fix whitespace, so we don't see these changes in PRs?

Reply:
We can 😄